### PR TITLE
Ft contribute linkl

### PR DESCRIPTION
**Pull request by mosesmulumba**
Added a contribute word in the landingfooter to link to https://github.com/crane-cloud.io

### CODE REVIEW

File: src/components/LandingFooter/index.js [hunk 1]
```diff
@@ -114,6 +115,13 @@ const LandingFooter = () => (
                   >
                     Blog
                   </a>
+                  <a
+                    href="https://github.com/cranecloud.io"
```
Comment (neelxie) [inline src/components/LandingFooter/index.js:119]: @mosesmulumba replace that with  https://github.com/crane-cloud

Comment (khalifan-kfan) [reply src/components/LandingFooter/index.js]: isn't blog supposed to be on a different line than contribute @mosesmulumba

Comment (khalifan-kfan) [reply src/components/LandingFooter/index.js]: image

**Review by neelxie — APPROVED** [review]
LGTM

**Pull request merged**